README.md

```diff
@@ -374,10 +374,12 @@ __IMPORTANT__
 Middleware **must** return either the result of the `next` call _or_ a promise/data structure to short circuit the stack with a response. They are mutually exclusive. Do not call both. Do not fail to return one or the other.
 
 ### authorize
-An optional predicate for checking the users permission. The envelope is provided to the function and should return a boolean or a promise that resolves to a boolean indicating whether or not the user can perform the requested action.
+The `authorize` predicate was added to actions to allow for much more fine grained, explicit control of user authorization. By default, authorization checks are performed after _all_ middleware has run. To change this, you can provide the string "authorize" in place of a middelware call in the resource or action middleware and the check will be performed in place of the string in the stack.
 
 > Note: when present, this _overrides_ rather than augments any authorization check that would have been performed by a configured autohost auth library.
 
+> Important! authorization takes place at the _end_ of the middleware stack. This is so that the middleware has every opportunity to provide any necessary data on the envelope's context before evaluating the user's access level.
+
```

Comment:
This probably could be reworded slightly given the new ability to specify where in the MW stack it runs (or just incorporate this into the first paragraph in this section)